physcraper/__init__.py

```diff
@@ -1683,8 +1617,60 @@ def __init__(self, data_obj, ids_obj):
         self.backbone = False
         self.OToL_unmapped_tips()  # added to do stuff with un-mapped tips from OToL
         self.ids.ingroup_mrca = data_obj.ott_mrca  # added for mrca ingroup list
-        self.gb_not_added = []  # list of blast seqs not added
+        #self.gb_not_added = set()  # list of blast seqs not added
         self.del_superseq = {}  # items that were deleted bc they are superseqs, needed for assert statement
+        self.mrca_ott_list = ingroup_mrca
+        assert type(self.mrca_ott_list) in [int, list] or self.mrca_ott_list is None
+        self.mrca_ncbi_list = set()  # corresponding ids for mrca_ott list
+        if self.mrca_ott_list is not None:
+            self.get_ncbi_mrca()
+        # self.mrca_ncbi_of_all = ids_obj.ott_to_ncbi[data_obj.ott_mrca]
+        if len(self.mrca_ncbi_list) == 0:
+            # self.mrca_ncbi = set()
+            self.mrca_ncbi_list.add(ids_obj.ott_to_ncbi[data_obj.ott_mrca])
+        debug(self.mrca_ncbi_list)
+
+
+
+    def get_ncbi_mrca(self):
+        """ get the ncbi tax ids from a list of mrca ott ids.
+        """
+        if type(self.mrca_ott_list) is not int:
+            for ott_id in self.mrca_ott_list:
+                ncbi_id = self.ottid_to_ncbiid(ott_id)
```

Comment:
We can update this to use the ids.ott_to_ncbi dictionary and throw an error if the ottid isn't in the dictionary. They all should be there, and it will be good to know if any aren't!

Reply:
Line 1640 is using what you are suggesting. I  added an else to write something to file if ott_id is not in the list

Reply:
Sounds good. We can cut out the taxomachine function entirely there too. If the input open tree dictionary is correct, then there shouldn't be any difference between what we're getting online versus locally. If there are things that are able to map through taxomachine but not locally, then we (I) need to fix the local dictionary map. So we should just throw an error.